### ISSUE TITLE
Test Android 13 Accessibility API's restriction

### ISSUE THREAD

**Issue by ferBonnin** [microsoft/accessibility-insights-for-android-service#158]
**To Reproduce**
Use the service and client  in Android 13

## Expected behavior
Service should still work

## Additional context
We might need to add a new step to AI4Android's install flow that uses an adb command to set the new "ACCESS_RESTRICTED_SETTINGS" permission to "allow"

**Comment by ferBonnin** [comment]
Peter tried this out with 13 R4, results are not consistent. Should check if inconsistency is due to R4, 13 or something else?

**Comment by peterdur** [comment]
@ferBonnin I looked again this week, still seeing intermittent results (sometimes screenshot appears, sometimes not) with API 33 (Tiramisu/Android 13). In parallel Madalyn was looking at #172; both are about missing screenshots, so reasonable to think that there may be related. Madalyn suggested there might be a race condition somewhere?

**Comment by ferBonnin** [comment]
closing as duplicate of #172, lets investigate them together
